src/ruby/yast/convert.rb

```diff
@@ -37,11 +37,9 @@ module Convert
     # @!method  self.to_locale(        object )
     #   @return        [String, nil]  *object*, or `nil` if it is not a String
     Ops::SHORTCUT_TYPES.each do |type|
-      eval <<END, binding, __FILE__, __LINE__ + 1
-        def self.to_#{type}(object)
-          convert object, :from => "any", :to => "#{type}"
-        end
-END
+      define_singleton_method("to_#{type}") do |object|
+        convert object, :from => "any", :to => "#{type}"
```

Comment:
While we're at this, can we simplify `"#{type}"` to `type`, which is a plain String?